PhotDM.tex

```diff
@@ -1565,9 +1571,16 @@ \subsection{Interrelation between Pogson and Asinh magnitudes}
 when b=0, it is recommended to use different implementation conversion codes for both types of magnitudes as a general implementation both for Pogson (b=0) and asinh (b>0) could easily produce numerical infinites during the evaluation.
 \par
 
+\section{Data Model VODML documentation}
 
-\section{Data Model Summary}
+The VODML description of this model is available at \\ \url{http://ivoa.net/xml/}.
```

Comment:
Please point to the actual VO-DML file.  The tricky question is: Do you want to link to "latest version 1" (which would be practical) or "exactly version 1.1" (which would arguably be more correct).  I have no strong opinions either way.